bin/swagger_style.py

```diff
@@ -43,13 +47,23 @@ def main() -> int:
     result = swagger_to.style.perform(swagger=swagger, typedefs=intermediate_typedefs, endpoints=endpoints)
 
     if result:
+        if args.with_line_number:
+            result.sort(key=lambda complaint: complaint.line)
+
         print("Style checks failed: \n")
-        if args.verbose:
-            for cmpl in result:
-                print("{}: {}: \"{}\"".format(cmpl.where, cmpl.message, cmpl.what.replace('\n', ' ')))
-        else:
-            for cmpl in result:
-                print("{}: {}".format(cmpl.where, cmpl.message))
+        for complaint in result:
+            complaint_str = ''
+            if args.with_line_number:
+                complaint_str += "{}:{} ".format(str(swagger_path.resolve()), complaint.line)
```

Comment:
Are you sure you need `resolve()` here? IMO, the user does not care if it's a link and it might be confusing to see the linked path instead of the link path.

Reply:
I included it so that the output of swagger_style could be copy-pasted anywhere and possibly generate links to the file and line number of interest. What I can surely avoid to do, though, is to call `resolve()` in each loop.

Reply:
Mind that `resolve()` makes the path absolute and resolves all the symlinks in the path (please see https://docs.python.org/3/library/pathlib.html#pathlib.Path.resolve). 

I assume the user would be very surprised to see the symlinks resolved especially if s/he is not even aware of them (which is sometimes the point of the symlinks in the first place :smiley: ). Analogously, if I run swagger_style in a directory, I don't want to see the absolute paths. This is also what mypy is doing in the ouptut. To see the absolute paths, you can always supply the absolute path as the command-line argument.

Reply:
Good point. Will fix it. (I also just realized that the unit tests would fail with absolute paths :dagger: )